test/hotspot/jtreg/compiler/lib/verify/Verify.java

```diff
@@ -202,83 +310,200 @@ private static void checkEQimpl(MemorySegment a, MemorySegment b, String context
         throw new VerifyException("MemorySegment value mismatch.");
     }
 
+    /**
+     * Verify that the content of two MemorySegments is identical. Note: we do not check the
+     * backing type, only the size and content.
+     */
```

Comment:
Probably a copy-paste error. Should be updated for exceptions.

Reply:
Good catch, updated it!